metrics/models.go

```diff
@@ -274,8 +300,7 @@ type FailuresMetadata struct {
 // loading the entity, for backward compatibility.
 type FailuresMetadataLegacy struct {
 	TestRunsMetadataLegacy
-	// Override the TestRuns to be omitted by datastore.
-	TestRuns shared.TestRuns `json:"test_runs,omitempty" datastore:"-"`
+	BrowserName string `json:"browser_name"`
```

Comment:
Why do we have a `BrowserName` here?

Reply:
It was incorrectly dropped in a previous PR (and is not part of the TestRunsMetadata class)